vignettes/tutorial-create-your-first-rhino-app.Rmd

```diff
@@ -59,8 +62,14 @@ There are two things you need to know when choosing this way of initializing you
 
 1. Rhino will __not__ change your working directory. You need to either
 open a new R session in your new application directory or manually change the working directory.
-2. Rhino relies on options added to `.Rprofile` file.
-The most robust way to make sure it was correctly sourced is to simply restart R session.
+2. Rhino relies on options added to the projects `.Rprofile` file.
+The most robust way to make sure it was correctly sourced is to simply restart the R session.
+
+```r
```

Comment:
This solves only the first point. I suggest moving it between 1 and 2 so it is more clear.